### PR TITLE
added titles of pages

### PR DESCRIPTION
**Pull request by Nikhilatvocabustudy**

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -47,7 +47,14 @@ const hashTitles = {
...
     "#search": "Browse Sets",
-    "#account": "My Account"
+    "#account": "My Account",
+    "#Privacy Policy": "Privacy Policy",
+    "Discord":"Discord",
+    "Feedback": "Feedback"
+    "Nightly Build": "Nightly Build"
+    "Leaderboard": "Leaderboard"
+    "Donate": "Donate"
+    "Youtube": "Youtube"
```
Comment (grimsteel) [inline src/js/index.js:57]: I feel like there's something wrong with every line here
1. Privacy policy is it's own separate page so it shouldn't be here
2. Feedback is not a page
3. Nightly build is not a page (where are you getting these things from??)
4. Leaderboard is not a page (I literally removed it > 1 month ago!)
5. The other ones should be exactly like the hash in index.pug

**Review by grimsteel — CHANGES_REQUESTED** [review]

File: src/js/index.js (hunk 1 above)
Comment (grimsteel) [reply src/js/index.js:57]: Also, that's not even valid JavaScript

Comment (Nikhilatvocabustudy) [reply src/js/index.js:57]: sorry I did this quite rushed, so I didn't really stop to look at it

File: src/js/index.js [hunk 2]
```diff
@@ -47,7 +47,10 @@ const hashTitles = {
     "#saved-sets": "Saved Sets",
     "#admin": "Admin Portal",
     "#search": "Browse Sets",
-    "#account": "My Account"
+    "#account": "My Account",
+    "#Discord":"Discord",
+    "#Donate": "Donate"
+    "#Youtube": "Youtube"
```
Comment (grimsteel) [inline src/js/index.js:53]: Looks good, just make the hashes lowercase and make sure there are commas after every element that is not the last one
```suggestion
    "#discord":"Discord",
    "#donate": "Donate",
    "#youtube": "Youtube"
```

**Review by grimsteel — CHANGES_REQUESTED** [review]

File: src/js/index.js [hunk 3]
```diff
@@ -47,7 +47,10 @@ const hashTitles = {
     "#saved-sets": "Saved Sets",
     "#admin": "Admin Portal",
     "#search": "Browse Sets",
-    "#account": "My Account"
+    "#account": "My Account",
+    "#discord":"Discord",
+    "#donate": "Donate",
+    "#youtube": "Youtube"
```
Comment (grimsteel) [inline src/js/index.js:53]: Oops - I didn't catch this:
You have to add the credits page too
```suggestion
    "#youtube": "Youtube",
    "#credits": "Credits"
```

**Review by grimsteel — CHANGES_REQUESTED** [review]

File: src/js/index.js (hunk 1 above)
Comment (PrakmO) [reply src/js/index.js]: Youtube is not a page

> I feel like there's something wrong with every line here
> 
> 1. Privacy policy is it's own separate page so it shouldn't be here
> 2. Feedback is not a page
> 3. Nightly build is not a page (where are you getting these things from??)
> 4. Leaderboard is not a page (I literally removed it > 1 month ago!)
> 5. The other ones should be exactly like the hash in index.pug

File: src/js/index.js (hunk 2 above)
Comment (PrakmO) [reply src/js/index.js]: But YouTube is not a page

File: src/js/index.js (hunk 3 above)
Comment (PrakmO) [reply src/js/index.js:53]: But YouTube isn't a page.... it's on Discord

**Comment by PrakmO** [comment]
ok nvm to all my comments about Youtube not being a page I saw PR #255

**Review by grimsteel — APPROVED** [review]

**Pull request merged**